### PR TITLE
0x5C4 is a valid product id for DualShock ZCT1E under Mac + Bluetooth

### PR DESCRIPTION
**Pull request by jimon**
Fixes 1286449

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 
 - Fixed precompiled layouts such as `FastKeyboard` leading to build time regressions with il2cpp (case 1283676).
+- Fixed DualShock support on Mac (case 1286449).
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Very unclear what that means. Better (also with link to public case):

> Fixed PS4 controller not recognized on Mac when connected over Bluetooth ([case 1286449](https://issuetracker.unity3d.com/issues/input-system-dualshock-4-zct1e-dualshock-2-v1-devices-are-not-fully-recognised-over-bluetooth)).

File: Assets/Tests/InputSystem/Plugins/DualShockTests.cs [hunk 2]
```diff
@@ -158,6 +158,18 @@ public void Devices_SupportsDualShockAsHID_WithJustPIDAndVID()
...
+        {
+            interfaceName = "HID",
+            capabilities = new HID.HIDDeviceDescriptor
+            {
+                vendorId = 0x54C,
+                productId = 0x5C4,
+            }.ToJson()
+        });
+
+        Assert.That(device, Is.AssignableTo<DualShockGamepad>());
```
Comment (Rene-Damm) [inline Assets/Tests/InputSystem/Plugins/DualShockTests.cs]: Recommend using `[TestCase]` for this one.

```
    [Test]
    [Category("Devices")]
    [TestCase(0x54C, 0x9CC)]
    [TestCase(0x54C, 0x5C4)]
    public void Devices_SupportsDualShockAsHID_WithJustPIDAndVID(int vendorId, int productId)
    {
        var device = InputSystem.AddDevice(new InputDeviceDescription
        {
            interfaceName = "HID",
            capabilities = new HID.HIDDeviceDescriptor
            {
                vendorId = vendorId,
                productId = productId,
            }.ToJson()
        });

        Assert.That(device, Is.AssignableTo<DualShockGamepad>());
    }
```

Comment (jimon) [reply Assets/Tests/InputSystem/Plugins/DualShockTests.cs]: fixed

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 1 above)
Comment (jimon) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: fixed

**Pull request merged**